src/api.coffee

```diff
@@ -153,24 +153,24 @@ exports.delete_resources = (public_ids, callback, options = {}) ->
   resource_type = options["resource_type"] ? "image"
   type = options["type"] ? "upload"
   uri = ["resources", resource_type, type]
-  call_api("delete", uri, _.extend({"public_ids[]": public_ids}, api.only(options, "keep_original", "invalidate")), callback, options)
+  call_api("delete", uri, _.extend({"public_ids[]": public_ids}, api.only(options, "keep_original", "invalidate", "transformations")), callback, options)
```

Comment:
`transformations` should be an array serialised as eager transformation. I don't see it here.

Also, Please add tests.

Reply:
Transformations are formatted on caller side and just retransferred to Clodinary there, so no data transformations required. We'll check how UT could be added for this case.